CMakeLists.txt

```
@@ -78,7 +78,7 @@ if (LIBCMAES_BUILD_PYTHON)
   find_package (PythonInterp)
   find_package (PythonLibs)
   if (NOT TARGET Python::Module AND PYTHONLIBS_FOUND)
-    add_library (Python::Module UNKNOWN IMPORTED)
+    add_library (Python::Module SHARED IMPORTED)
     target_include_directories (Python::Module INTERFACE ${PYTHON_INCLUDE_DIRS})
     set_target_properties (Python::Module PROPERTIES IMPORTED_LOCATION
                                                      ${PYTHON_LIBRARIES})
```

The cause for #212 is that target_include_directories can only be used with IMPORTED targets since cmake 3.11, which I missed. The change from UNKNOWN to imported above is probably not needed, but also should not hurt.

```suggestion
    set_target_properties (
      Python::Module
      PROPERTIES IMPORTED_LOCATION ${PYTHON_LIBRARIES}
                 INTERFACE_INCLUDE_DIRECTORIES ${PYTHON_INCLUDE_DIRS})
```